src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -736,13 +737,20 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
     protected function getContainerLoader(ContainerInterface $container)
     {
         $locator = new FileLocator($this);
-        $resolver = new LoaderResolver(array(
+        $loaders = array(
             new XmlFileLoader($container, $locator),
             new YamlFileLoader($container, $locator),
             new IniFileLoader($container, $locator),
             new PhpFileLoader($container, $locator),
             new ClosureLoader($container),
-        ));
+        );
+
+        // DirectoryLoader only appears in DI 2.7
+        if (class_exists('Symfony\\Component\\DependencyInjection\\Loader\\DirectoryLoader')) {
```

Comment:
@laurentb I don't think it's worth it. People using Standard Edition will get both components in right versions. Anyone using DependencyInjection and HttpKernel components standalone, is already maintaining versions himself in his composer.json. I fail to see a scenario when he'd like to bump one component's version bot not the other.